editoast/src/views/projects.rs

```diff
@@ -227,20 +233,20 @@ struct ProjectPatchForm {
     /// The id of the image document
     pub image: Option<i64>,
     #[schema(max_length = 255)]
-    pub tags: Option<Vec<String>>,
+    pub tags: Option<Vec<Option<String>>>,
 }
 
 impl ProjectPatchForm {
     fn into_project(self, project_id: i64) -> Project {
         Project {
-            id: Some(project_id),
-            name: self.name,
-            description: self.description,
-            objectives: self.objectives,
-            funders: self.funders,
-            budget: self.budget,
-            image: Some(self.image),
-            tags: self.tags,
+            id: project_id,
+            name: self.name.unwrap_or_default(),
+            description: self.description.unwrap_or_default(),
+            objectives: self.objectives.unwrap_or_default(),
+            funders: self.funders.unwrap_or_default(),
+            budget: self.budget.unwrap_or_default(),
+            image: self.image,
+            tags: self.tags.unwrap_or_default(),
```

Comment:
This doesn't work, sending a patch for the `name` field will reset all other fields to their default value.

We should use the `ProjectChangeset` to patch an existing project.